### PR TITLE
SAM-2559 improved Grading and Feedback ui

### PR DESCRIPTION
**Pull request by ern**
Before:
before

After:
after

### CODE REVIEW

File: samigo/samigo-app/src/webapp/js/authoring.js [hunk 1]
```diff
@@ -389,16 +400,19 @@ function disableAllFeedbackCheckTemplate(feedbackType)
 	}
 }
 
-$(document).ready(function(){
+$(window).load( function() {
```
Comment (ottenhoff) [inline samigo/samigo-app/src/webapp/js/authoring.js]: I'm unclear on the logic here. Isn't document.ready the preferred way here?

http://learn.jquery.com/using-jquery-core/document-ready/

Comment (ern) [inline samigo/samigo-app/src/webapp/js/authoring.js]: So document.ready runs when the DOM is ready but window.load is when the whole page is loaded including the content.

I noticed much smoother transitions while using window.load vs document.ready. Also noticed some javascript issues while updating the date pickers state.

Comment (ottenhoff) [inline samigo/samigo-app/src/webapp/js/authoring.js]: Okay, makes sense. +1

Comment (ern) [inline samigo/samigo-app/src/webapp/js/authoring.js]: Forgot to mention that window.load is fired later than document.ready
http://stackoverflow.com/questions/8396407/jquery-what-are-differences-between-document-ready-and-window-load

**Pull request merged**